site/gatsby-site/src/components/checklists/CheckListForm.js

```diff
@@ -0,0 +1,481 @@
+          severity: SEVERITY.danger,
+          error,
+        });
+      }
+    }
+  };
+
+  isSubmitting; // We should use this instead
+  const [saveStatus, setSaveStatus] = useState(null);
+
+  const [risksLoading, setRisksLoading] = useState(false);
+
+  const [allPrecedents, setAllPrecedents] = useState([]);
+
+  const searchTags = [...values['tags_goals'], ...values['tags_methods'], ...values['tags_other']];
+
+  useEffect(() => {
+    searchRisks({ values, setFieldValue, setRisksLoading, setAllPrecedents, setSaveStatus });
+  }, [values['tags_goals'], values['tags_methods'], values['tags_other']]);
+
+  const oldSetFieldValue = setFieldValue;
+
+  setFieldValue = (key, value) => {
+    oldSetFieldValue(key, value);
+    submitForm();
+  };
+
+  const debouncedSetFieldValue = useRef(debounce(setFieldValue, 1000)).current;
+
+  const removeRisk = (findFunction) => {
+    setFieldValue(
+      'risks',
+      values.risks.filter((risk) => !findFunction(risk))
+    );
+  };
+
+  const changeSort = (sortFunction) => (event) => {
+    event.preventDefault();
+    setFieldValue('risks', values.risks.sort(sortFunction));
+  };
+
+  const updateRisk = (risk, attributeValueMap) => {
+    const updatedRisks = [...values.risks];
+
+    const updatedRisk = updatedRisks.find((r) => risksEqual(r, risk));
+
+    for (const attribute in attributeValueMap) {
+      if (attribute != 'precedents') {
+        updatedRisk.generated = false;
+      }
+      updatedRisk[attribute] = attributeValueMap[attribute];
+    }
+
+    setFieldValue('risks', updatedRisks);
+    submitForm();
+  };
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <Header>
+        <LocalizedLink to="/apps/checklists" className="text-lg">
+          <Trans>Risk Checklists</Trans>
+        </LocalizedLink>
+        <HeaderRow>
+          <h1>
+            <EditableLabel
+              title={values.name}
+              onChange={(event) => debouncedSetFieldValue('name', event.target.value)}
+              textClasses="text-2xl"
+              iconClasses="text-lg vertical-align"
+            />
+          </h1>
+          <HeaderControls>
+            <SavingIndicator {...{ saveStatus }} />
+            <Button color="light" onClick={() => alert('Coming soon')}>
+              <Trans>Subscribe</Trans>
+            </Button>
+            <DeleteButton type="button" onClick={() => confirmDeleteChecklist(values.id)}>
+              <Trans>Delete</Trans>
+            </DeleteButton>
+            <ExportDropdown checklist={values} />
+          </HeaderControls>
+        </HeaderRow>
+      </Header>
+      <Info>This feature is in development. Data entered will not be retained.</Info>
+      {/*<Info className="my-4">
+        <Trans>
+          Describe the system under investigation. Apply machine-readable tags to surface risks
+          associate with likewise-tagged incidents in the database. These are the basis of a risk
+          checklist.
+        </Trans>
+      </Info>*/}
```

Comment:
Please remove the commented code or add a note explaining the reason to keep it.

Reply:
A fault of the GitHub review API, we discussed these comments a bit. It's okay for the prototype while there are open design questions for the UI, but should be removed going forward.

https://github.com/responsible-ai-collaborative/aiid/pull/1852#discussion_r1333568842

Reply:
I went ahead and remove them anyway. It won't be hard to add them back in if necessary.